cppForSwig/Makefile.am

```diff
@@ -154,6 +153,7 @@ ARMORYCOMMON_SOURCE_FILES = Accounts.cpp \
 	Wallets.cpp \
 	WebSocketClient.cpp \
 	WebSocketMessage.cpp \
+	DBUtils.cpp \
```

Comment:
Why this file was put here? I guess they were sorted alphabetically before.

Reply:
Hi Sergey, for some odd reason I did not get an email for your comment.

The reason that file is there is that I noticed while working on the cmake code that ArmoryCommon now depends on it, while it was in ArmoryCLI before.

I just moved it into the correct spot alphabetically.